types/texzilla/index.d.ts

```diff
@@ -0,0 +1,103 @@
+// Type definitions for texzilla 0.9
+// Project: https://github.com/fred-wang/TeXZilla/
+// Definitions by: Michal Grňo <https://github.com/m93a>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export default TeXZilla;
```

Comment:
This package appears to contain its exports directly rather than through a default export -- `require("texzilla")` is `{ setDOMParser: [Function], ... }`, not `{ default: { setDOMParser: [Function], ... } }`.
So, I this would be better if you removed this line and the `declare namespace TeXZilla` wrapper and just exported all functions directly with `export function`. Then when importing the package you should be able to use named imports `import { setDOMParser, ... } from "texzilla";`.

Reply:
I've tried both versions in my program, and they both seem to work in runtime. However, I've changed the definition to the one you proposed, as it seems to be the one intended by the package's developers.